docs/asyncapi.yaml

```diff
@@ -254,10 +255,11 @@ components:
           items:
             type: object
             required:
-              - accountCode
               - description
               - schemeCode
               - value
+              - deliveryBody
+              - marketingYear
```

Comment:
I think there's a few other required missing from the list - should they be added?

Reply:
Fine by me :)